app/components/public/add-to-calender.hbs

```diff
@@ -9,7 +9,16 @@
       {{/if}}
       ({{this.timezone}})
       <br>
-      <a href={{this.calendarUrl}} target="_blank" rel="noreferrer nofollow">{{t 'Add to Google Calendar'}}</a>
+      <UiDropdown class="pointing">
+	<a href="#">{{t 'Add to Calendar'}}</a>
+	<div class="menu">
+	  {{#each this.calendarUrls as |calendar|}}
+	    <a href={{calendar.url}} class="item">
```

Comment:
I have no idea why you remove target blank and rel nofollow noreferrer. There is a reason why I added that